packages/app-tron/tests/01.create/index.ts

```diff
@@ -0,0 +1,25 @@
+import { MockDeviceConnection } from '@cypherock/sdk-interfaces';
```

Comment:
Need to add tests for all functions of `app-tron`

Reply:
test cases added